src/api/app/components/workflow_run_header_component.rb

```diff
@@ -0,0 +1,18 @@
+class WorkflowRunHeaderComponent < WorkflowRunRowComponent
```

Comment:
This inheritance is a bit weird... A header component and a row component shouldn't have much in common.

It shows that `WorkflowRunRowComponent` does quite a lot and it could be split up in multiple components. It doesn't need to happen now, but I just wanted to point it out.

Reply:
@dmarcoux created a card in the backlog to track this https://trello.com/c/o4CGrUEL/2478-split-up-workflowrunrowcomponent